privacy_evaluator/attacks/attack_interface.py

```diff
@@ -0,0 +1,30 @@
+import abc
+
+
+class Attack_Interface():
+
+    """Initilizes the Attack Class.
+    :param model: the model to be attacked.
+    :type model: ART-Classifier
+    :param training_data: training data of the model.
+    :type test_accuracy: TODO: What Type?
+    :param test_data: test data of the model.
+    :type test_data: TODO: What Type?
+    """
+
+    def __init__(self, model, training_data, test_data):
+        self.model = model
+        self.training_data = training_data
```

Comment:
Is the training_data and test_data a tuple (data, labels)?

Reply:
This is actually code from team 1 and they fixed it as far as we know.